tiles/src/test/java/com/protomaps/basemap/names/ScriptTest.java

```diff
@@ -0,0 +1,85 @@
+package com.protomaps.basemap.names;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ScriptTest {
+
+  @Test
+  void singleScript() {
+    assertEquals("LATIN", Script.getScript("Berlin"));
+    assertEquals("ARABIC", Script.getScript("دبي"));
+    assertEquals("CYRILLIC", Script.getScript("Скопје"));
```

Comment:
Might add a comment this is for Russian and related languages?

Reply:
Added